Makefile

```diff
@@ -300,7 +300,6 @@ manifests:
 	mv ${TMPDIR}/cilium.io_ciliumidentities.yaml ./examples/crds/ciliumidentities.yaml
 	mv ${TMPDIR}/cilium.io_ciliumnodes.yaml ./examples/crds/ciliumnodes.yaml
 	mv ${TMPDIR}/cilium.io_ciliumlocalredirectpolicies.yaml ./examples/crds/ciliumlocalredirectpolicies.yaml
-	mv ${TMPDIR}/cilium.io_ciliumclusterwidelocalredirectpolicies.yaml ./examples/crds/ciliumclusterwidelocalredirectpolicies.yaml
```

Comment:
(Note for other reviewers, this is the Clusterwide LRP which was not implemented)